.github/workflows/build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   test:
     strategy:
       matrix:
-        suite: [unit, web, esm, cjs]
+        suite: [unit, web, esm, cjs, integration]
```

Comment:
fine to keep - but if we see issues with setup/tests we may want to remove as we continue to harden